README.md

```diff
@@ -25,10 +25,11 @@ By default, the local development server is running at `localhost:8000`.
 
 ## Configuring backend
 
-By default, Gatsby will try to connect to localhost:1337, where Strapi is expected to be running. If you want to use local Strapi, see the set up guide in the `tikweb-cms` repository. If you want to instead connect to the production version of Strapi, rename the `.env.example` file to `.env.development`, and put the following line in the file, if it is not there:
+By default, Gatsby will try to connect to localhost:1337, where Strapi is expected to be running. If you want to use local Strapi, see the set up guide in the `tikweb-cms` repository. If you want to instead connect to the production version of Strapi, rename the `.env.example` file to `.env.development`, and put the following content in the file, if it is not there:
 
 ```
 STRAPI_URL=https://tikweb-prod-app-cms.azurewebsites.net
+STRAPI_API_KEY=<api key here>
```

Comment:
We'll need a stable way of creating this token. By default they only last for like 7 days. Or maybe we should just create one with 20 years expiry and stick it in Secrets? :D

https://docs.strapi.io/developer-docs/latest/plugins/users-permissions.html#jwt-configuration

I'll create something for now so we can deploy this.